custom/logistics/api.py

```diff
@@ -255,5 +255,11 @@ def add_language_to_user(self, logistics_sms_user):
                 v = VerifiedNumber.by_phone(phone_number, include_pending=True)
                 if v:
                     v.delete()
-                user.save_verified_number(self.domain, phone_number, True,
-                                          logistics_sms_user.backend)
+                try:
+                    user.save_verified_number(self.domain, phone_number, True,
+                                              logistics_sms_user.backend)
+                except PhoneNumberInUseException:
+                    v = VerifiedNumber.by_phone(phone_number, include_pending=True)
+                    v.delete()
+                    user.save_verified_number(self.domain, phone_number, True,
+                                              logistics_sms_user.backend)
```

Comment:
I'm not sure this is a good workflow, since we could be deleting real numbers in use in the system for other purposes. do we have any way to know that we are just replacing a number that came over in a previous migration? @gcapalbo do you have any feedback on how best to handle this?